src/seedu/addressbook/AddressBook.java

```diff
@@ -439,7 +460,7 @@ private static String executeAddPerson(String commandArgs) {
      */
     private static String getMessageForSuccessfulAddPerson(String[] addedPerson) {
         return String.format(MESSAGE_ADDED,
-                getNameFromPerson(addedPerson), getPhoneFromPerson(addedPerson), getEmailFromPerson(addedPerson));
+                getNameFromPerson(addedPerson), getPhoneFromPerson(addedPerson), getEmailFromPerson(addedPerson), getDateOfBirthFromPerson(addedPerson));
```

Comment:
You've modified this function to return more information for the user. In this case, your regression testing will fail, and should be updated accordingly. In addition, you should also update the header comments so that the change is reflected.